chunk_downloader.go

```diff
@@ -357,7 +361,7 @@ func downloadChunkHelper(ctx context.Context, scd *snowflakeChunkDownloader, idx
 	}
 	bufStream := bufio.NewReader(resp.Body)
 	defer resp.Body.Close()
-	logger.Infof("response returned chunk: %v, resp: %v", idx+1, resp)
+	logger.Infof("response returned chunk: %v for URL: %v, resp: %v", idx+1, scd.ChunkMetas[idx].URL, resp)
```

Comment:
Lets not dump the response. Also do we want it in info? Do you foresee this becoming noisy?